### PR TITLE
oneflow.tensor 未翻译的部分

### PR DESCRIPTION
**Pull request by robin978**
[oneflow.Tensor.addcmul](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.addcmul) , [oneflow.Tensor.addcmul_](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.addcmul_) , [oneflow.Tensor.addmm](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.addmm) , 
[oneflow.Tensor.amax](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.amax) ，[oneflow.Tensor.amin](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.amin) ,
截屏2022-05-12 下午6 31 16

[oneflow.Tensor.asinh](https://start.oneflow.org/oneflow-api-cn/te
截屏2022-05-12 下午6 31 34
nsor.html#oneflow.Tensor.asinh) , 

[oneflow.Tensor.byte](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.byte) , 
截屏2022-05-12 下午6 31 51

[oneflow.Tensor.cosh](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.cosh) ,
截屏2022-05-12 下午6 32 10

[oneflow.Tensor.global_to_global](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.global_to_global) ,
截屏2022-05-12 下午6 34 53

[oneflow.Tensor.half](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.half) ,
截屏2022-05-12 下午6 35 59

[oneflow.Tensor.local_to_global](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.local_to_global) ,
截屏2022-05-12 下午6 37 36

[oneflow.Tensor.new_empty](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.new_empty) , 
截屏2022-05-12 下午6 38 12

[oneflow.Tensor.new_zeros](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.new_zeros) , 
截屏2022-05-12 下午6 38 35

[oneflow.Tensor.to_consistent](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.to_consistent) ,
截屏2022-05-12 下午6 40 59

 [oneflow.Tensor.view_as](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.view_as) ,
截屏2022-05-12 下午6 41 27

### CODE REVIEW

File: docs/source/cn/tensor.py [hunk 1]
```diff
@@ -1304,3 +1304,289 @@
...
+    Note:
+        这个张量必须是全局张量。
+
+        至少需要 placement 和 sbp 中的一个操作。
+
+        如果 placement 和 sbp 都与这个张量的 placement 和 sbp 相同，那么返回自己的这个张量。
+    
+    参数：
+        - **placement** (flow.placement, optional) - 返回全局张量的所需 placement，默认值：None。
+        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) - 返回全局张量的所需 sbp，默认值：None。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1383]:
```suggestion
        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) - 返回 global tensor 的所需 sbp，默认值：None。
```

File: docs/source/cn/tensor.py [hunk 2]
```diff
@@ -1304,3 +1304,289 @@
...
+
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.local_to_global,
+    """
+    Tensor.local_to_global(placement=None, sbp=None, *, check_meta=Ture) -> Tensor
+
+    从一个局部张量构造一个全局张量。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1429]:
```suggestion
    从一个 local tensor 构造一个 global tensor。
```

File: docs/source/cn/tensor.py [hunk 3]
```diff
@@ -1304,3 +1304,289 @@
...
+
+    Note:
+        这个张量必须是局部张量。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1432]:
```suggestion
        这个张量必须是 local tensor。
```

File: docs/source/cn/tensor.py [hunk 4]
```diff
@@ -1304,3 +1304,289 @@
...
+
+        placement 和 sbp 属性都需要。
+
+        返回的全局张量将该张量作为其在当前 rank 中的局部分量。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1436]:
```suggestion
        返回的 global tensor 将该张量作为其在当前 rank 中的 local tensor。
```

File: docs/source/cn/tensor.py [hunk 5]
```diff
@@ -1304,3 +1304,289 @@
...
+
+        通常数据是不会被广播的，但是当 sbp 为 ``oneflow.sbp.broadcast`` 时，rank 0 的数据将被广播到其他等级。
+    
+    参数：
+        - **placement** (flow.placement, optional) - 返回全局张量的所需 placement，默认值：None。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1441]:
```suggestion
        - **placement** (flow.placement, optional) - 返回 global tensor 的所需 placement，默认值：None。
```

File: docs/source/cn/tensor.py [hunk 6]
```diff
@@ -1304,3 +1304,289 @@
...
+        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) - 返回全局张量的所需 sbp，默认值：None。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1442]:
```suggestion
        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) - 返回 global tensor 的所需 sbp，默认值：None。
```

File: docs/source/cn/tensor.py [hunk 7]
```diff
@@ -1304,3 +1304,289 @@
...
+    关键参数：
+        - **check_meta** (bool, optional) - 注明当从局部张量构建全局张量时是否要检查元信息。只有设置为 False 时，每
```
Comment (robin978) [inline docs/source/cn/tensor.py:1444]:
```suggestion
        - **check_meta** (bool, optional) - 注明当从 local tensor 构建 global tensor 时是否要检查元信息。只有设置为 False 时，每
```

File: docs/source/cn/tensor.py [hunk 8]
```diff
@@ -1304,3 +1304,289 @@
...
+            个 rank 上输入的局部张量的形状和类型都相同，如果设置为 False，则加速执行 local_to_global。默认值：True。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1445]:
```suggestion
            个 rank 上输入的 local tensor的形状和类型都相同，如果设置为 False，则加速执行 local_to_global。默认值：True。
```

File: docs/source/cn/tensor.py [hunk 9]
```diff
@@ -1304,3 +1304,289 @@
...
+    """
+    Tensor.new_empty(*size, dtype=None, device=None, placement=None, sbp=None, requires_grad=False) -> Tensor
+
+    返回一个充满未初始化数据大小为 :attr:`size` 的张量。默认情况下，返回的张量具有与此张量相同的 :attr:`flow.dtype` 和 :attr:`flow.device`。
+
+    参数：
+        - **size** (int...) - 一个 list，tuple 或整数的 flow.Size，定义输出张量的形状。
+        - **dtype** (flow.dtype, optional) - 返回张量的所需类型，默认值如果为 None，则返回与此张量相同的 flow.dtype。
+        - **device** (flow.device, optional) - 返回张量的所需设备，默认值如果为 None，则返回与此张量相同的 flow.device。
+        - **placement** (flow.placement, optional) - 返回的全局张量的所需的 placement，默认值如果为 None，则返回的张量是使用参数 `device` 的本地张量。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1488]:
```suggestion
        - **placement** (flow.placement, optional) - 返回的 global tensor 的所需的 placement，默认值如果为 None，则返回的张量是使用参数 `device` 的 local tensor。
```

File: docs/source/cn/tensor.py [hunk 10]
```diff
@@ -1304,3 +1304,289 @@
...
+        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) -返回的全局张量所需的 sbp 描述，默认值如果为 None，则返回的张量是使用参数 `device` 的本地张量。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1489]:
```suggestion
        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) -返回的 global tensor 所需的 sbp 描述，默认值如果为 None，则返回的张量是使用参数 `device` 的 local tensor。
```

File: docs/source/cn/tensor.py [hunk 11]
```diff
@@ -1304,3 +1304,289 @@
...
+    """
+    Tensor.new_zeros(size=None, dtype=None, device=None, placement=None, sbp=None, requires_grad=False) -> Tensor
+
+    返回一个大小为 0 的张量。默认情况下，返回的张量与该张量具有相同的 flow.dtype 和 flow.device。
+
+    参数：
+        - **size** (int...) - 一个 list，tuple 或整数的 flow.Size，定义输出张量的形状。
+        - **dtype** (flow.dtype, optional) - 返回张量的所需类型，默认值如果为 None，则返回与此张量相同的 flow.dtype。
+        - **device** (flow.device, optional) - 返回张量的所需设备，默认值如果为 None，则返回与此张量相同的 flow.device。
+        - **placement** (flow.placement, optional) - 返回的全局张量的所需的 placement，默认值如果为 None，则返回的张量是使用参数 `device` 的本地张量。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1516]:
```suggestion
        - **placement** (flow.placement, optional) - 返回的 global tensor 的所需的 placement，默认值如果为 None，则返回的张量是使用参数 `device` 的 local tensor。
```

File: docs/source/cn/tensor.py [hunk 12]
```diff
@@ -1304,3 +1304,289 @@
...
+        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) -返回的全局张量所需的 sbp 描述，默认值如果为 None，则返回的张量是使用参数 `device` 的本地张量。
```
Comment (robin978) [inline docs/source/cn/tensor.py:1517]:
```suggestion
        - **sbp** (flow.sbp.sbp or tuple of flow.sbp.sbp, optional) -返回的 global tensor 所需的 sbp 描述，默认值如果为 None，则返回的张量是使用参数 `device` 的 local tensor。
```

**Comment by robin978** [comment]
https://github.com/Oneflow-Inc/oneflow-api-cn/issues/36#issuecomment-1122050606
目前还遇到了这个问题

**Review by Chenqll — APPROVED** [review]

**Pull request merged**